lms/djangoapps/course_api/blocks/transformers/tests/test_student_view.py

```diff
@@ -1,16 +1,17 @@
 """
 Tests for StudentViewTransformer.
 """
+import ddt
 
 # pylint: disable=protected-access
-
-from openedx.core.lib.block_structure.factory import BlockStructureFactory
+from openedx.core.djangoapps.content.block_structure.factory import BlockStructureFactory
```

Comment:
I think this should be using the old location, and you should not include the new `openedx/core/djangoapps/content/block_structure/factory.py` file.

Reply:
Didn't know there was an old one! Let me take care of that and remove all the added files.